app/src/main/AndroidManifest.xml

```diff
@@ -12,9 +12,8 @@
         android:roundIcon="@drawable/ic_logo"
         android:supportsRtl="true"
         android:theme="@style/Theme.EFood">
-        <activity
-            android:name=".ui.splash.SplashActivity"
-            android:theme="@style/NoActionBarTheme">
+        <activity android:name=".ui.splash.SplashActivity"
+            android:exported="true">
```

Comment:
_*tham khảo*_
Nếu được thì lần sau em tạo ra 1 cái image = paint/photoshop có logo và text như mong muốn 
rồi làm theo cách này thì tối ưu hơn 
sau em muốn load hay init gì trong splash thì thời gian calculate sẽ chính xác hơn 
https://shishirthedev.medium.com/the-right-way-to-implement-a-splash-screen-in-android-acae0e52949a